### DISCUSSION TITLE
行数が長すぎてコメント更新できないときにエラーを表示して欲しい

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#733] [要望・アイデア・不具合]
コメントにコードを書いたあとUMLを追記しようとしたのですが、行数が400行近くになったためか、更新を押してもページ遷移せずエラーも表示されません。
もし行数が長すぎるのが原因であれば、「コメントが長すぎます」などのエラーを表示していただけないでしょうか。

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご不便をおかけして申し訳ありません。
コメントには文字数の上限があるため、恐らく文字数を超えてしまったのではないかと思います。
エラーメッセージを表示するよう改善いたします！🙇

**Reply by yuta0709** [reply]
@shiracamus 
お待たせいたしました🙇‍♂️
コメントの文字数が制限を超えた場合に、エラーメッセージが表示されるよう改善いたしました。
この度はご不便をおかけして申し訳ありませんでした。
今後ともQiitaをよろしくお願いいたします！
